src/frontend/src/crypto/mnemonic.ts

```diff
@@ -0,0 +1,11 @@
+import { blobFromUint32Array } from "@dfinity/agent";
+import { entropyToMnemonic } from "bip39";
+
+/**
+ * @returns A random BIP39 mnemonic with 256 bits of entropy.
+ */
+export function generateMnemonic(): string {
+  const entropy = new Uint32Array(32);
+  crypto.getRandomValues(entropy);
+  return entropyToMnemonic(blobFromUint32Array(entropy).toString("hex"));
```

Comment:
I think we should explicitly set the word list here, I'm not sure if it picks something locale dependent by default?

Reply:
I believe it's English by default, but I made it explicit to make it clearer.